src/cpp/fastdds/publisher/DataWriterImpl.hpp

```diff
@@ -771,6 +773,19 @@ class DataWriterImpl : protected rtps::IReaderDataFilter
 
 private:
 
+    /**
+     * (Re)configures the deadline timer:
+     *  Create once, parked with a huge interval (idle).
+     *  In case of deadline period ∞ cancel it, for 0 warn and notify once; set counts to max and
+     *  for values >0 store period.
```

Comment:
```suggestion
     *  In case of infinite deadline period cancel it, for 0 warn and notify once (with max counts), and
     *  for non-infinite positive values store period.
```